JSON_Schemas/Animal_Health_and_Treatments/iscWithdrawalType.json

```diff
@@ -11,6 +11,10 @@
             "type":"string",
             "format": "dateTime",
             "description": "Date till when the withholding applies"
+        },
+        "market" : {
```

Comment:
Andrew Cooke to review the spreadsheet. I thought we had considered using a scheme approach.